projectroles/tests/test_templatetags.py

```diff
@@ -725,3 +725,184 @@ def test_get_sidebar_links_timeline_view(self):
                     self.assertEqual(app['active'], True)
                 else:
                     self.assertEqual(app['active'], False)
+
+    def test_get_user_links_home(self):
```

Comment:
Also one very small thing: for consistency, let's name these test methods in a kind of a hierarchical fashion. `get_links_home_user()` and `get_links_home_superuser()` etc is better, as the modifying things come last.